examples/w3c-vc/specification.yml

```diff
@@ -1,7 +1,6 @@
 user_claims:
```

Comment:
should this file be removed with this change to use arf-pid? also the one in w3c-vc_for_slide_deck?

Reply:
I'd really like to remove unused stuff unless there's some compelling reason to keep it around.

Reply:
I would like to have a place to have examples that are not referenced in the spec - you think danielfett/sd-jwt is a better option?

Reply:
I'm not sure TBH but maybe. They aren't even examples but descriptor files that input into danielfett/sd-jwt tooling to generate examples. So living closer to danielfett/sd-jwt might be a better option. I was just looking to remove unused stuff here to keep this repo more 'clean' and make it easier to maintain. But we can leave it for now in order to move forward w/ getting this example in and publishing.